.ci_support/environment-old.yml

```diff
@@ -3,7 +3,7 @@ channels:
 dependencies:
 - dill =0.3.1.1
 - gitpython =3.1.23
-- h5io =0.1.2
+- h5io =0.1.8
```

Comment:
The ability to operate on an already open h5py file object was added by @pmrv  in version 0.1.8 https://github.com/h5io/h5io/releases/tag/h5io-0.1.8